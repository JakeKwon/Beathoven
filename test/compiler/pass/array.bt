int [] arr;
arr = [];
arr = [2, 4, 6, 8, 10];
print(arr[3]);
arr[3] = 9;
print(arr[3], " array is good\n");