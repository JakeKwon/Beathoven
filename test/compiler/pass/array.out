89 array is good
